src/components/shared/Navbar.tsx

```diff
@@ -60,7 +60,10 @@ export const Navbar = ({
           <MenuItem to="/community">COMMUNITY</MenuItem>
           <MenuItem to="https://github.com/opendatahub-io">GITHUB</MenuItem>
           <MenuItem to="/docs/quick-installation/">
-            GET STARTED
+            GET STARTED V1
```

Comment:
To prevent confusion, we should leave this unchanged as `GET STARTED` with a link to the existing operator installation until the next release where we are ready for all users to migrate to the new operator for deploying all ODH components

Reply:
reverted